app/services/project-skills-list.js

```diff
@@ -6,13 +6,22 @@ import recordsList from 'code-corps-ember/utils/records-list';
 export default Service.extend({
   store: service(),
 
+  project: null, // set typically in afterModel
+
   projectSkills: alias('project.projectSkills'),
 
   add(skill) {
     let { store, project } = getProperties(this, 'store', 'project');
     return store.createRecord('project-skill', { project, skill }).save();
   },
 
+  addWithoutSave(skill) {
+    let { store, project } = getProperties(this, 'store', 'project');
+    let projectSkill = store.createRecord('project-skill', { project, skill });
+    project.get('skills').pushObject(skill);
+    return projectSkill;
```

Comment:
I don't like that this is necessary to do now.

Reply:
Considering it's a `many-to-many` now I'm not sure there's a need for this service.

When creating the project, we assign to the `many-to-many` directly.
When updating the project, we create individual `projectSkill` records. We _could_ assign and remove from the `many-to-many` to, but at the moment, we don't and I'm not sure it would be an improvement.

As far as I know, those are our only two scenarios for working with project skills. Considering that, this doesn't really dry up our code. It appears to do so, but under the hood, it's just trying to fit two different chunks of code together.